openwrt/Makefile

```diff
@@ -1,17 +1,24 @@
 include $(TOPDIR)/rules.mk
 
 PKG_NAME:=prince
-PKG_VERSION:=v0.3
-PKG_RELEASE:=3
+PKG_VERSION:=v0.3.3
+PKG_RELEASE:=4
```

Comment:
If i remember correctly the Releases number is resetted to one for each new version